sidechain/consensus/slots/src/slots.rs

```diff
@@ -208,11 +208,11 @@ mod tests {
 	}
 
 	fn test_block_with_time_stamp(timestamp: u64) -> SignedBlock {
+		let header = SidechainHeaderT::new(0, H256::random(), H256::random(), Default::default());
```

Comment:
We should introduce a header builder, just like the block builder. (for the parentchain, we already have a header builder)